### DISCUSSION TITLE
New Source Team Orange

### DISCUSSION THREAD

**Discussion by ChrisKoh83** [mampfes/hacs_waste_collection_schedule#1926] [Ideas]
Please add Team Orange as new source 

https://www.team-orange.info/index.html

**Comment by 5ila5** [comment]
Team orange is supported by the [Abfall.IO / AbfallPlus.de source,](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/abfall_io.md) I added it to the wizard and supported list in  #1934.

The Team orange App is supported by the [Apps by Abfall+ Source](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/app_abfallplus_de.md)
